pkg/controller/backup_controller_test.go

```diff
@@ -224,6 +244,43 @@ func TestProcessBackup(t *testing.T) {
 					mock.Anything, // actions
 				).Return(nil)
 
+				var location *v1.BackupStorageLocation
```

Comment:
What I would do here is always add two locations to the informer - `default` and `loc1`.  Then, keep the 3 test cases you have, but change the third to use location `loc2` (which won't exist).  Your verification can stay as-is but the setup will be more straightforward